src/components/views/analytics/SelectDisplayedUsersLimit.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { FormControl, Grid, InputLabel, MenuItem, Select } from '@mui/material';
+
+import { UserLimitOption } from '../../../types/chart';
+import { topMembersRangeOptions } from '../../../utils/chart';
+
+interface Props {
+  displayedUsersLimit: UserLimitOption;
+  setDisplayedUsersLimit: (val: UserLimitOption) => void;
+}
+const SelectDisplayedUsersLimit = ({
+  displayedUsersLimit,
+  setDisplayedUsersLimit,
+}: Props): JSX.Element => {
+  const { t } = useTranslation();
+  const handleChange = ({
+    target: { value },
+  }: {
+    target: { value: string };
+  }) => {
+    const memberRange = topMembersRangeOptions.find(
+      (ele) => ele.value === value
+    );
+    if (memberRange) {
+      setDisplayedUsersLimit(memberRange);
+    }
+  };
+  return (
```

Comment:
You should add some empty lines to segment your code and make it easier to read. Currently it is hard to parse the different sections.